### DISCUSSION TITLE
Block stories that need to be paid to see

### DISCUSSION THREAD

**Discussion by icariantk** [dailydotdev/daily#1436] [Feature Requests]
There are website that not just an account is needed, but a paid account. Could be usefull to discriminate those sites from a custom feed.

**Comment by nimrodkra** [comment]
Paywalled content is not allowed on our platform. We have a system in
place that should detect and automatically remove paywalled content but
it's not perfect. Once in a while it might miss a paid post. In such cases
the best thing you can do is to report the post. That way it helps our
system learn and you can get reputation points.
